Jenkinsfile-onfinality

```diff
@@ -136,6 +143,13 @@ pipeline {
                 }
             }
         }
+        stage('Send Notification to telegram') {
+            steps { 
+                sh """
+                   curl -s -X POST https://api.telegram.org/bot${TELEGRAM_BOT_ID}/sendMessage -d chat_id=${TELEGRAM_CHAT_ID} -d text='${currentBuild.fullDisplayName} - ${currentBuild.currentResult}. Details ${BUILD_URL}'
```

Comment:
we already have function for it in library, use it

Reply:
sure,  I will try to use.